README.md

```diff
@@ -445,6 +446,12 @@ If you wanted to bulk export a subset of the filtered results without having to
 const results = await manager.runCustomFilterQuery({whiteList: ['id'], pageSize: 10000});
 ```
 
+### Get the raw elasticsearch query derived from the current filters
+
+```typescript
+const rawEsQuery = await manager.getCurrentEsQuery();
```

Comment:
Can you also add this to api -> manager -> methods:

[https://github.com/social-native/elastic-composer/blob/master/README.md#methods](https://github.com/social-native/elastic-composer/blob/master/README.md#methods)